src/topology/metric_space/basic.lean

```diff
@@ -1802,11 +1819,25 @@ lemma nndist_pi_le_iff {f g : Πb, π b} {r : ℝ≥0} :
   nndist f g ≤ r ↔ ∀b, nndist (f b) (g b) ≤ r :=
 by simp [nndist_pi_def]
 
+lemma nndist_pi_lt_iff {f g : Πb, π b} {r : ℝ≥0} (hr : 0 < r) :
+  nndist f g < r ↔ ∀b, nndist (f b) (g b) < r :=
```

Comment:
```suggestion
  nndist f g < r ↔ ∀ b, nndist (f b) (g b) < r :=
```

Reply:
This is deliberate to match the style of the line above. I don't want to fix both because that just makes forward-porting more annoying.

Reply:
I hereby allow you to break post universal quantifier whitespace consistency in this file 😉